### PR TITLE
Update easy-connect / Realtek WiFi

### PR DESCRIPTION
**Pull request by JanneKiiskila**
Enable REALTEK_RTL8195AM WiFi.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,24 +172,26 @@ With Thread, you can change the operating mode of the client from the default ro
 
 #### Supported boards
 
-* UBLOX_EVK_ODIN_W2. Check instructions for compilation [here](#compile-configuration-for-odin-wifi).
+* UBLOX_EVK_ODIN_W2. Check instructions for compilation [here](#compile-configuration-for-odin-and-realtek-wifi).
```
Comment (peknis01) [inline README.md]: Read the [instructions for compilation](#compile-configuration-for-odin-and-realtek-wifi). And the same for the other Realtek case.

**Comment by JanneKiiskila** [comment]
rebuild

**Comment by JanneKiiskila** [comment]
Rebased and force pushed.

**Comment by teetak01** [comment]
Handled as part of https://github.com/ARMmbed/mbed-os-example-client/pull/327

**Pull request closed without merging**